cmd/eval.go

```diff
@@ -882,15 +860,14 @@ func generateOptimizedBundle(params evalCommandParams, asBundle bool, filter loa
 		WithTarget(params.target.String()).
 		WithAsBundle(asBundle).
 		WithOptimizationLevel(params.optimizationLevel).
-		WithOutput(buf).
+		WithOutput(bytes.NewBuffer(nil)).
 		WithEntrypoints(params.entrypoints.v...).
 		WithRegoAnnotationEntrypoints(true).
 		WithPaths(paths...).
 		WithFilter(filter).
 		WithRegoVersion(params.regoVersion())
 
-	err := compiler.Build(context.Background())
-	if err != nil {
+	if err := compiler.Build(context.Background()); err != nil {
```

Comment:
[nit] Let's use `c.Context()` if it's anywhere close. Fine to keep as-is if it isn't.

Reply:
What would `c` in that be? I don't see anything like that in scope.